doc/scenarioParametersCustomization.md

```diff
@@ -4,149 +4,134 @@ This document describes advanced methods to customize the scenario parameters vi
+You will then be able to declare parameters with your custom `varType` in the YAML solution description, and the webapp
 will automatically generate the associated components.
 
-### Extended _varTypes_
+### Extended `varType`
 
 In some cases you may want to define a **sub-type** to have a distinction between several parameters with the same
-_varType_. For instance, the specific _varType_ `%DATASETID%` is used for all dataset parts, but you may need to display
-different component in the webapp for some file parameters (e.g; a file upload and a table that users can edit).
+`varType`. For instance, the specific `varType` `%DATASETID%` is used for all dataset parts, but you may need to display
+different component in the webapp for some file parameters (e.g; a file upload vs. a table that users can edit).
 
-To define an extended var type, you must add a `subType` property in your parameter configuration:
-
-- `subType` (optional): define a varType suffix to handle this parameter as a custom varType
+To define an extended var type, you must set the `options.subType` property in your parameter configuration.
 
 Example:
 
-```
-const PARAMETERS = {
-  my_dataset_part_paramaete: {
-    connectorId: 'C-xxxxxxxxxx',
-    defaultFileTypeFilter: '.zip,.csv,.json,.xls,.xlsx',
-    description: 'Initial stock dataset part',
-    subType: 'TABLE'
-  },
-};
+```yaml
+parameters:
+  - id: 'volume_unit'
+    labels:
+      en: 'Volume unit'
+      fr: 'Unité de volume'
+    varType: 'enum'
+    defaultValue: 'LITRE'
+    options:
+      enumValues:
+        - key: 'LITRE'
+          value: 'L'
+        - key: 'BARREL'
+          value: 'bl'
+        - key: 'CUBIC_METRE'
+          value: 'm³'
+      subType: 'RADIO'
 ```
 
-If the _varType_ of the parameter above is `%DATASETID%`, then the extended _varType_ for this parameter will be
-`%DATASETID%-TABLE`.
+This sub-type will be concatenated after the `varType` value. For instance, if the `varType` of a parameter is
+`%DATASETID%` and its sub-type is `TABLE`, then the extended `varType` for this parameter will be `%DATASETID%-TABLE`.
 
-You will then be able to use this extended _varType_ in the four mapping files (see section "Define a custom
-_varType_"). For any of those files, if the extended _varType_ of a parameter is not found in the mapping dict, the
-webapp will fallback on its _varType_. This means that if you want to keep the same behavior for an extended _varType_
-as for the orgiinal _varType_, you can omit to declare the extended varType in the mapping dicts.
+You will then be able to use this extended `varType` in the four mapping files (see section "Define a custom
+`varType`"). For any of those files, if the extended `varType` of a parameter is not found in the mapping dict, the
+webapp will fallback on its `varType`. This means that if you want to keep the same behavior for an extended `varType`
```

Comment:
I have just learned this rule: **fallback** is a noun  and **fall back** is a verb :rofl: